### PR TITLE
DTM-1248 DTM-1249 postgresql connectors

### PR DESCRIPTION
**Pull request by FDoKE**

### CODE REVIEW

**Review by vbir94 — DISMISSED** [review]

File: kafka-postgres-avro/src/main/java/io/arenadata/kafka/postgres/avro/model/AvroQueryResultRow.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        this.schema = schema;
+        this.inputRow = inputRow;
+    }
+
+    @Override
+    public void put(int i, Object v) {
+        if (outputRow.size() == i) {
+            outputRow.add(v);
+        } else {
+            throw new IllegalArgumentException(String.format("Неправильный порядок: %d, %s", i, v));
```
Comment (artem-krutov) [inline kafka-postgres-avro/src/main/java/io/arenadata/kafka/postgres/avro/model/AvroQueryResultRow.java]: Should it be in English, and also should we give variables more informative name?

File: kafka-postgres-reader/src/main/java/io/arenadata/kafka/postgres/reader/service/impl/PublishServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Service
+public class PublishServiceImpl implements PublishService {
```
Comment (artem-krutov) [inline kafka-postgres-reader/src/main/java/io/arenadata/kafka/postgres/reader/service/impl/PublishServiceImpl.java]: Please, don't 😃
I bet if you drop this `PublishService` and get rid of this `Impl` for each and every single interface, nothing happens. No errors nor exceptions. Spring won't fail and life will become simpler.

Comment (artem-krutov) [reply kafka-postgres-reader/src/main/java/io/arenadata/kafka/postgres/reader/service/impl/PublishServiceImpl.java]: Please try to delete the interface and call this service class `PublishService`, i.e. delete this ugly `Impl`. Good time to check if spring proxies could deal with that.

**Review by artem-krutov — DISMISSED** [review]
Looks good 👍
Some configurations might be common for reader and writer – does it make sense to make it shared?

**Review by artem-krutov — APPROVED** [review]

**Pull request merged**